Misc/NEWS.d/next/Core and Builtins/2021-12-11-11-36-48.bpo-46045.sfThay.rst

```diff
@@ -0,0 +1 @@
+Do not use POSIX semaphores on NetBSD
```

Comment:
The changelog also needs an explanation why POSIX semaphores cannot be used on NetBSD. Is there an issue with NetBSD kernel or do we need extra quirks to support NetBSD flavor of semaphores?

Reply:
Sorry, I don't know that.
I've tried using gdb to look at the python processes that are forked by the three mentioned tests, but there are a lot of them. Can you suggest a simple example where Python would use semaphores that I could use to file a NetBSD bug report?